octokit/resources.py

```diff
@@ -226,7 +196,5 @@ def fetch_resource(self, method, *args, **kwargs):
     response = self.session.send(prepared_req)
 
     schema = self.parse_schema(response)
-    self.rels = self.parse_rels(response)
-    self.session.last_response = response
 
-    return Resource(self.session, schema=schema, name=humanize(self.name))
+    return Resource(self.session, response=response, schema=schema, name=humanize(self.name))
```

Comment:
Yep me too. I tried to do this though and I ended up in an endless recursion in `__getattr__`. I saw you passed response in your branch though so maybe you know how we could deal with that!